onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -144,7 +144,7 @@ void BasicBackend::PopulateConfigValue(ov::AnyMap& device_config) {
     device_config.emplace(ov::hint::inference_precision("f32"));
   }
   if (global_context_.precision_str.find("ACCURACY") != std::string::npos &&
-      global_context_.device_type == "GPU") {
+      global_context_.device_type.find("GPU") != std::string::npos) {
```

Comment:
this change LGTM. it should handle any '.x' values after 'GPU' (GPU, GPU.0, GPU.1)..

Our current logic would fail for OV 2025.0.
@jatinwadhwa921 can you also please update line 148 to:

if (((OPENVINO_VERSION_MAJOR == 2024) && (OPENVINO_VERSION_MINOR > 1)) || (OPENVINO_VERSION_MAJOR > 2024))

Reply:
LGTM